package.json

```diff
@@ -45,9 +45,9 @@
     "react-bootstrap": "^1.6.3",
     "react-dom": "^17.0.2",
     "react-icons": "^4.2.0",
+    "react-native": "^0.65.1",
```

Comment:
Please uninstall this, it added 4k lines to your package-lock file 😂 . `npm uninstall react-native`